tests/TestStubs.php

```diff
@@ -125,43 +24,27 @@ public static function setUpBeforeClass()/* The :void return type declaration th
         self::$mutedProblems = new MutedProblems();
     }
 
-
-    public function constantProvider()
-    {
-        foreach (ReflectionStubsSingleton::getReflectionStubs()->constants as $constant) {
-            yield "constant {$constant->name}" => [$constant];
-        }
-    }
-
     /**
-     * @dataProvider constantProvider
+     * @dataProvider \TestData\Providers\ReflectionTestDataProviders::constantProvider
      */
-    public function testConstants(stdClass $constant)
+    public function testConstants(PHPConst $constant)
     {
         $constantName = $constant->name;
         $constantValue = $constant->value;
```

Comment:
We can actually try to compare const values in stubs and in runtime. In case, there are not a lot of false positives that should be useful.

Reply:
Well, I've tried. And the result is next: Tests: 1528, Failures: 88. Most of the failures are platform dependent but taking in account that it's just 88 of 1528 constants probably it makes sense to add this test

Reply:
The numbers look indeed ok to include the check. We can turn it off anytime if we will find that most new constants are platform dependant.

Reply:
Done. You can find the list of failing constants in mutedProblems.json